README.md

```diff
@@ -0,0 +1,61 @@
+# About workshop
+
+In this workshop we will explore what Firebase Functions are and setup project to be able to write those functions in Kotlin. To get there we will touch some other interesting things like Realtime Database and TypeScript. 
+
+
+### Requirements:
+
+ * [Firebase account](https://console.firebase.google.com/)
+ * Code editor for JavaScript/TypeScript (my choice is [VS Code](https://code.visualstudio.com/))
+ * [IntelliJ Idea community edition](https://www.jetbrains.com/idea/download/)
+
+
+### What are we building:
+
+> **Ultimate Currency converter**
+> 
+> Let's make a backend for the ultimate currency convertion app - user enters price in euro and backend converts it to other currencies.
+
+Backend will work as follows: 
+ * Client app pushes new "conversion" with euro value into table
+ * Function gets triggered and picks up this value and calculates other currencies
+ * Data gets written back to same object with a timestamp
+
+Firebase Realtime Database structure:
+``` js
+{
+    "rates" : {
+        "usd" : 1.175,
+        "czk" : 25.88,
+        "pln" : 4.303,
+        "rub" : 68.52
```

Comment:
lvl?

Reply:
There is no exchange rate for it for quiet a while.